src/utils.js

```diff
@@ -287,6 +303,40 @@ genotet.utils.keySet = function(collection) {
 };
 
 /**
+ * Validates email address.
+ * @param {string} email
+ * @return {boolean}
+ */
+genotet.utils.validateEmail = function(email) {
```

Comment:
These functions are a bit redundant. You can do
```js
genotet.utils.validateRegex = function(str, regex) {
   return RegExp(regex).test(str);
}
```
Length check can be encoded in regex, such as /\w{MIN_LEN}/.
However, since this regex test line is so short, it may not be necessary to create a utils function for this (though I don't object to have one either).